.github/workflows/mobilecoin-workflow-dev-update-block-version.yaml

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+#
+# MobileCoin Core projects - Reusable Workflow - update consensus nodes in a development namespace.
+
+name: mobilecoin-workflow-dev-update-block-version
+
+on:
+  workflow_call:
+    inputs:
+      namespace:
+        description: "Target Namespace"
+        type: string
+        required: true
+      block_version:
+        description: "block_version"
+        type: string
+        required: true
+        default: '0'
```

Comment:
Do we want to set a default here?  Thought is that once we move off  version zero is there a potential that this does not get bumped and we deploy back to version 0?  Is that even possible or would it error out because the block version is lower than what is currently deployed?

Reply:
Deploying back to a previous version would probably make a big mess. You're probably right, removing the version default is a good idea,